tests/Unit/Domain/Test_SizeOfElement.cpp

```diff
@@ -148,18 +146,17 @@ void test_3d(const std::array<double, 4>& times_to_check) noexcept {
       std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>
       functions_of_time = make_single_expansion_functions_of_time();
 
-  const auto check_helper =
-      [&functions_of_time, &logical_to_grid_map,
-       &times_to_check](const auto& grid_to_inertial_map) noexcept {
-        std::array<double, 3> element_size{};
-        for (const double time : times_to_check) {
-          size_of_element(make_not_null(&element_size), logical_to_grid_map,
-                          grid_to_inertial_map, time, functions_of_time);
-          const auto expected_size_of_element =
-              make_array(0.0125, 0.85, 0.125) * (UseMovingMesh ? time : 1.0);
-          CHECK_ITERABLE_APPROX(element_size, expected_size_of_element);
-        }
-      };
+  const auto check_helper = [&functions_of_time, &logical_to_grid_map,
```

Comment:
the helper is almost identical in all 3 places, can it be factored into one with an additional argument?

Reply:
The helper is removed in a subsequent commit in this PR

Reply:
okay, I won't make you factor it then 😛